.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Prerequisites
+
+Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+<hr>
+
+# Expected Behaviour
+
+Please describe the behaviour you are expecting
+
+# Current Behaviour
+
+What is the current behaviour?
+
+<hr>
+# Failure Information (for bugs 🪲)
```

Comment:
Is this a header? Not displaying on my as such in the preview mode.

Reply:
It was! Thanks for spotting that.

I needed to add a line between the <hr> and the header for markup to pick up.

Took the chance to make this section a bit clearer by separating the message on reporting bugs. Hopefully the guidance on filling these out is easier to follow now